### DISCUSSION TITLE
mac 暂时解决方案

### DISCUSSION THREAD

**Discussion by Naoki19880107** [getlantern/lantern-forum-cn#330] [Ideas]
下载https://github.com/getlantern/lantern-binaries-archive里的6.9.10，覆盖安装，重新激活专业版，在把7.8.3装回去，启动后，手动去开网页代理，在网卡那里设置，然后就不要关了，关了就在重复一遍上面操作

**Comment by MaxMKMax** [comment]
@Naoki19880107 这位朋友谢谢您的分享!
